server/src/infra/migrations/1709763765506-AddExtensionToOriginalFileName.ts

```diff
@@ -0,0 +1,20 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddExtensionToOriginalFileName1709763765506 implements MigrationInterface {
```

Comment:
@jrasm91 Migration is added

@mertalev Does the query look ok here to append the file extension to the original name?